.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - run: cargo install cargo-hack
       - run: git config user.name "github-runner" && git config user.email "<>"
-      - run: git rebase --exec 'cargo hack --clean-per-run --feature-powerset --exclude-features alloc --exclude-features default --version-range 1.82.. --clean-per-version check --locked' --exec 'cargo clean' --root
+      - run: git rebase --exec 'cargo hack --clean-per-run --feature-powerset --exclude-features alloc --exclude-features default --exclude-features unsafe-pinned --version-range 1.82.. --clean-per-version check --locked' --exec 'cargo clean' --root
```

Comment:
please leave it in the tests, that's what I intended the `CONFIG_RUSTC_HAS_UNSAFE_PINNED` to be for